src/httpbeast.nim

```diff
@@ -54,6 +54,10 @@ type
     domain*: Domain
     numThreads: int
     loggers: seq[Logger]
+    reusePort*: Option[bool]
+      # when false, will fail if --threads is passed or if 2
+      # processes bind to same address/port. We use an Option to avoid
+      # breaking code that relies on `Settings(port: port)`
```

Comment:
hm, I think we can live with this breakage. `initSettings` should be used anyway.

Reply:
it's in the wild though... in fact, 1st version of this didn't use an Option and it broke some code. Other than that, i agree that initSettings should be used